src/bo4e_cli/commands/pull.py

```diff
@@ -39,7 +54,10 @@ def pull(
         typer.Option(
             help="A GitHub Access token to authenticate with the GitHub API. "
             "Use this if you have problems with the rate limit. "
-            "Alternatively, you can set the environment variable GITHUB_ACCESS_TOKEN.",
+            "It is encouraged to set the environment variable GITHUB_ACCESS_TOKEN instead to prevent"
+            "accidentally storing your token into the shell history. "
+            "If you have the GitHub CLI installed and the token can't be found in the environment variables, "
+            "the token will be fetched from the GitHub CLI (if you are logged in). Uses `gh auth token`.",
```

Comment:
hatte gerade den Gedanken ob man für Powershell, bash/zsh noch die befehle hinterlegt wie man die Umgebungsvariable setzt. Bin aber unsicher ob es das wirklich braucht.

Reply:
Nah. Jeder Developer kann es im Zweifel in <1min ergoogeln.